datasets/SCCS/codes.csv

```diff
@@ -9023,409 +9023,409 @@ SCCS1713,NA,Missing data,NA
-SCCS1721,NA,Missing data,NA
-SCCS1721,1,absence of rich (original code 10),absence of rich
-SCCS1721,2,"presence of rich, no information on numbers (original code 20)","presence of rich, no information on numbers"
-SCCS1721,3,few rich (original code 21),few rich
-SCCS1721,4,many rich (original code 22),many rich
-SCCS1722,NA,Missing data,NA
-SCCS1722,0,"absence of rich, variable 1717 coded 1 (original code 88)","absence ofrich, variable 1717 coded 1"
-SCCS1722,1,"presence of rich, no information on sources of wealth (original code 10)","presence of rich, no information on sources of wealth"
-SCCS1722,2,acquired wealth of land (through buying or skill) (original code 11),acquired wealth of land
-SCCS1722,3,inheritance of land (original code 12),inheritance of land
-SCCS1722,4,acquired wealth of cattle (through buying or skill) (original code 13),acquirred wealth of cattle
-SCCS1722,5,inheritance of cattle (original code 14),inheritance of cattle
-SCCS1722,6,acquired wealth (means of production other than cattle or land) (original code 15),acquired wealth
-SCCS1722,7,inherited wealth (means of production other than cattle or land) (original code 16),inherited wealth
-SCCS1722,8,more than one of the above (original code 177),more than one of the above
-SCCS1723,NA,Missing data,NA
-SCCS1723,1,absence of poor (original code 10),absence of poor
-SCCS1723,2,"presence of poor, no information on percentage (original code 20)","presence of poor, no information on percentage"
-SCCS1723,3,few poor (original code 21),few poor
-SCCS1723,4,many poor (original code 22),many poor
-SCCS1724,NA,Missing data,NA
-SCCS1724,1,no dispossessed (original code 10),no dispossessed
-SCCS1724,2,"presence of dispossessed, no information on percentage (original code 20)","presence of dispossessed, no information on percentage"
-SCCS1724,3,few dispossessed (original code 21),few dispossessed
-SCCS1724,4,many dispossessed (original code 22),many dispossessed
-SCCS1725,NA,Missing data,NA
-SCCS1725,0,no need for expansion: variable 1716 coded as 1 (original code 88),no need for expansion: variable 1716 coded as 1
-SCCS1725,1,peaceful territorial expansion impossible (original code 10),no need for expansion: variable1716 coded as 1
-SCCS1725,2,"peaceful territorial expansion possible, no information on quality of land (original code 20)",peaceful territorial expansion impossible
-SCCS1725,3,access to land of good quality (original code 21),access to land of good quality
-SCCS1725,4,access to land of restricted quality (original code 22),access to land of restricted quality
-SCCS1726,NA,Missing data,NA
+SCCS1721,NA,missing data,missing data
+SCCS1721,10,absence of rich,absence of rich
+SCCS1721,20,"presence of rich, no information on numbers","presence of rich, no information on numbers"
+SCCS1721,21,few rich,few rich
+SCCS1721,22,many rich,many rich
+SCCS1722,NA,missing data,missing data
+SCCS1722,88,absence of rich (variable 11 coded ´10´),absence of rich (variable 11 coded ´10´)
```

Comment:
@kirbykat I realize I have to adapt the references to other SCCS variables in the descriptions here.